[Term Entry] JavaScript Dates .setUTCDate() method (#2598)
Codecademy/docs: content/javascript/concepts/dates/terms/setUTCDate/setUTCDate.md

---
Title: '.setUTCDate()'

> Description: 'Changes the day to a Date object in Coordinated Universal Time (UTC).'

Reviewer: ```suggestion
Description: 'Changes the day of a Date object in Coordinated Universal Time (UTC).'
```

Subjects:
  - 'Web Development'
Tags:
  - 'Date'
  - 'Methods'
  - 'Time'
CatalogContent:
  - 'paths/web-development'
  - 'introduction-to-javascript'
---

> The **`.setUTCDate()`** method changes the day to a Date object.

Reviewer: ```suggestion
The **`.setUTCDate()`** method changes the day of a Date object in Coordinated Universal Time (UTC)..
```

## Syntax

```pseudo
myDate.setUTCDate(day)
```

> The `.setUTCDate()` method receives a day argument representing a number between 1 to 31.

Reviewer: ```suggestion
The `.setUTCDate()` method receives a `day` argument representing a number between 1 to 31.
```

## Example

> In this example, we create a Date object in a variable called `myDate`. We then use `.setUTCDate()` to modify the day to 10.

Reviewer 2: ```suggestion
In this example, a Date object is created in a variable called `myDate`. `.setUTCDate()` is then used to modify the day to 10.
```
try not to use pronouns

Reviewer: In particular we avoid using first- or second-person pronouns in Docs.
